### ISSUE TITLE
Problem with logging internal deprecations

### ISSUE THREAD

**Issue by samuelszabo** [LordSimal/cakephp-sentry#1]
We have this setup:
- cakephp 4.4.11
- cakephp-sentry 1.0.3
- sentry 3.15.0

## Problem
In case of deprecation in the internal php function (as `implode(['array'], 'glue')`), there is an exception fired, and deprecation is not logged.

Exception:
`User Error: [TypeError] Argument 2 passed to Sentry\FrameBuilder::buildFromBacktraceFrame() must be of the type int, string given, called in vendor/sentry/sentry/src/StacktraceBuilder.php on line 64`

## Cause
This is caused by CakePHP's debugger, which is using '??' as a default for unknown lines 
https://github.com/cakephp/cakephp/blob/4.x/src/Error/Debugger.php#L440
and sentry is expecting an int.
https://github.com/getsentry/sentry-php/blob/94e1a8ed9fcfb41fec740c9a973a81bfffcd14ea/src/StacktraceBuilder.php#L52

## Solution
I suggest replacing line `'??'` with `0` in trace before pushing to Sentry.
https://github.com/LordSimal/cakephp-sentry/blob/main/src/Http/SentryClient.php#L203

**Comment by robertpustulka** [comment]
This came up again with CakePHP 4.5

My failing stack trace:
```
Stack Trace:
- /app/vendor/cakephp/cakephp/src/Error/ExceptionTrap.php:285
- [internal]:??
- [main]:
```

**Comment by LordSimal** [comment]
Yup, missed that release. See https://github.com/LordSimal/cakephp-sentry/releases/tag/1.0.5
